package.json

```diff
@@ -45,8 +45,10 @@
     "lodash": "4.17.11",
     "node-sass": "4.11.0",
     "prettier": "1.16.4",
+    "react-loader-spinner": "2.3.0",
     "react-select": "2.4.2",
     "react-table": "6.9.2",
+    "redux-persist": "^5.10.0",
```

Comment:
Remove that sign `^` here too.

Reply:
create an issue to refactor dependencies.

Reply:
As I see all dependencies are without that sign `^`, so leaving it is not consistent.

Reply:
A bot updates this.

Reply:
I see, thank you for your comment.